src/TinyGsmClientSIM7000.h

```diff
@@ -569,101 +568,169 @@ class TinyGsmSim7000 : public TinyGsmModem<TinyGsmSim7000>,
  protected:
   bool modemConnect(const char* host, uint16_t port, uint8_t mux,
                     bool ssl = false, int timeout_s = 75) {
-    if (ssl) { DBG("SSL not yet supported on this module!"); }
-
     uint32_t timeout_ms = ((uint32_t)timeout_s) * 1000;
-    sendAT(GF("+CIPSTART="), mux, ',', GF("\"TCP"), GF("\",\""), host,
-           GF("\","), port);
-    return (1 ==
-            waitResponse(timeout_ms, GF("CONNECT OK" GSM_NL),
-                         GF("CONNECT FAIL" GSM_NL),
-                         GF("ALREADY CONNECT" GSM_NL), GF("ERROR" GSM_NL),
-                         GF("CLOSE OK" GSM_NL)));
+
+    sendAT(GF("+CACID="), mux);
+    if (waitResponse(timeout_ms) != 1) return false;
+
+    if (ssl) {
+      sendAT(GF("+CSSLCFG=\"sslversion\",0,3"));  // TLS 1.2
```

Comment:
The `0` should be a mux.
like:
```
sendAT(GF("+CSSLCFG=\"sslversion\","),mux,",3");
```

Reply:
No, I think the 0 is the PDP context identifier, not the socket (mux) identifier.  It looks to be correct as-is.